package.json

```
@@ -6,11 +6,11 @@
   "type": "commonjs",
   "types": "types/index.d.ts",
   "scripts": {
-    "coverage": "tap --cov --coverage-report=html test.js",
+    "coverage": "c8 node --test",
```

Can we add coverage requirement
```suggestion
    "coverage": "c8 --100 node --test",
```